scripts/validate-notebooks.ps1

```diff
@@ -64,10 +70,10 @@ function Validate {
     $ErrorActionPreference = 'Continue'
     if ($env:SYSTEM_DEBUG -eq "true") {
         # Redirect stderr output to stdout to prevent an exception being incorrectly thrown.
-        jupyter nbconvert $CheckNotebook --execute --to html --ExecutePreprocessor.timeout=120 --log-level=DEBUG 2>&1 | %{ "$_"}
+        jupyter nbconvert $CheckNotebook --execute --to html --ExecutePreprocessor.startup_timeout=300 --ExecutePreprocessor.timeout=180 --log-level=DEBUG 2>&1 | %{ "$_"}
```

Comment:
The cell execution time for Q# notebooks shouldn't be affected, but for the Python notebooks, the potential time increase would show up as part of the cell execution that contains `import qsharp`.

These may be more aggressive than necessary. I will try reducing these both to 120 to see if I get any failures. (My thought was that it's better to set it too high rather than have the risk of spurious CI failures. What do you prefer?)

Reply:
There are going to be some CI failures regardless - the transient failures I see are on tasks that should take 5-10 seconds to execute, not 120, so it's rather unlikely that they'll complete in 180 seconds but not in 120. I think I'd prefer to keep the limits lower so that we get a signal if our performance suddenly degrades (and the frequency of failures increases).